frontend/src/components/containers/ContainersDetailContent.js

```diff
@@ -22,6 +22,7 @@ const pageStyle = {
 
 const tabsStyle = {
   marginTop: 8,
+  overflow: 'auto'
```

Comment:
This might be a good opportunity to clean up the copy/pasted `tabsStyle` and `tabStyle` stuff, and define the style once in a shared file, rather than adding `overflow:auto` everywhere.